scripts/lib_build_antithesis_images.sh

```diff
@@ -65,6 +66,9 @@ function build_antithesis_images {
  --build-arg GO_VERSION=${go_version}\
  --build-arg BUILDER_IMAGE_TAG=${image_tag}\
  --build-arg BUILDER_WORKDIR=${builder_workdir}"
+  if [[ -n "${avalanchego_commit:-}" ]]; then
```

Comment:
Relative bash novice here, and feel free to ignore either way, but is the :- necessary given that we already have `local avalanchego_commit=${10:-}` above?